front_end/StereoMatch[] matches
geometry_msgs/Point32[] Triangulated
std_msgs/Int32 nLeft
std_msgs/Int32 nRight
std_msgs/Int32 loweRatio
